MachineLearning/TensorBoard/tensorboard.md

```diff
@@ -21,11 +21,10 @@ For RL it is useful to visualize metrics such as:
  * Estimated Q-values.
  * Algorithm-specific metrics (e.g. entropy for PPO).
  
-To visualize results from Tensorboard, first `cd` to the `ray_results` directory:
+To visualize results from Tensorboard, first `cd` to the directory where your results reside. E.g., if you ran experiments using `ray`, then do the following:
 ```
 cd ~/ray_results/
 ```
-Every ML/RL experiment generates a subdirectory.
 
 Begin by activating an Anaconda environment:
```

Comment:
Sounds good. I suggest dropping the Conda bit too just to keep it generic except the tensorboard portion. Once that is removed can merge.

Reply:
The thing is that to use Tensorboard, you need to activate the Conda environment (at least that's how I know it, unless one would install Tensorboard at the `base` env)

Reply:
What do you think?

Reply:
You could use it from a pip install in userspace or in a container images as well. I think it is best to keep it as 'agnostic' to the install as possible. I'll defer to you though if you think it is important we can leave it in and merge as is.

Reply:
Oooh I see, I didn't know that you can use alternative methods like that to activate Tensorboard. In that case, I could rephrase it as:

```There are three main methods for activating Tensorboard:
* If you included Tensorboard installation in an Anaconda environment, activate it and proceed
* You can also install Tensorboard in userspace using `pip install`
* Install it using container images

Then [...]
```

P.S. is it possible you provide me with examples for the pip install userspace or container images so that I include them in the tutorial as well?

Reply:
Below is an example of running it from a container
```
ml singularity-container
singularity pull docker://tensorflow/tensorflow
singularity run tensorflow_latest.sif
tensorboard --logdir=. --port 6006 --bind_all
```

Python user
```
pip install tensorboard --user
tensorboard --logdir=. --port 6006 --bind_all
```
Yes having the different options listed like you have would be good.

Reply:
Great, thanks! I will perform the changes asap. When done, take a look and lmk if it's ok